Lib/test/test_pyrepl/test_unix_console.py

```diff
@@ -303,3 +303,15 @@ def test_getheightwidth_with_invalid_environ(self, _os_write):
             self.assertIsInstance(console.getheightwidth(), tuple)
             os.environ = []
             self.assertIsInstance(console.getheightwidth(), tuple)
+
+    def test_mac_with_invalid_environ(self, _os_write):
+        # gh-128636 for macOS
+        if sys.platform == "darwin":
+            console = UnixConsole(term="xterm")
```

Comment:
```suggestion
    @unittest.skipUnless(sys.platform == "darwin")
    def test_mac_with_invalid_environ(self, _os_write):
        # gh-128636 for macOS
```

and dedent the rest.